services/user-login-manager/app/db.py

```diff
@@ -38,6 +38,7 @@ class UserSQL(Base):
     first_name: Mapped[str] = mapped_column(nullable=False)
     last_name: Mapped[str] = mapped_column(nullable=False)
     email: Mapped[str] = mapped_column(nullable=False)
+    role: Mapped[str] = mapped_column(nullable=True)
```

Comment:
User role should not be nullable, right?

Reply:
Good catch, changed.